### ISSUE TITLE
plugin.video.sbs - (from bs4 import BeautifulSoup - ImportError: No module named bs4)

### ISSUE THREAD

**Issue by tkf144** [xbmc-catchuptv-au/repo#63]
Hi guys,

Firstly, thanks for your hard work on these addons.

Recently started using plugin.video.sbs 1.1.1 with Kodi 15.3 on OpenELEC on an RPI3. Initially installed from the AussieAddons repo. Auto-update fired night-before-last and 1.1.2 came down. The addon has since failed to launch and re-installing from repo has no effect.

On launching the addon:
```
ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
     - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
    Error Type: <type 'exceptions.ImportError'>
    Error Contents: No module named bs4
    Traceback (most recent call last):
      File "/storage/.kodi/addons/plugin.video.sbs/default.py", line 29, in <module>
        import categories  # noqa: E402
      File "/storage/.kodi/addons/plugin.video.sbs/resources/lib/categories.py", line 19, in <module>
        import comm
      File "/storage/.kodi/addons/plugin.video.sbs/resources/lib/comm.py", line 28, in <module>
        from bs4 import BeautifulSoup
    ImportError: No module named bs4
    -->End of Python script error report<--
```

Downloading and installing the 1.1.2 .zip from the plugin.video.sbs repo results in successful installation of the BeautifulSoup4 addon but failure of the SBS addon (`ERROR: CAddonInstallJob[plugin.video.sbs]: could not read addon description of special://home/addons/plugin.video.sbs`).

However, adding `version="4.3.2"` to the bs4 import in addon.xml and installing from zip results in the successful installation of both BS4 and SBS, and the SBS addon launches and plays streams.

I'm not a Kodi dev so I've just been poking around trying to get it going. In trying to figure it out, I noticed [this line still refers to BS3](https://github.com/xbmc-catchuptv-au/repo/blob/7f93033ff8ce62af1f204f0899e9fc5058a3b55c/addons.xml#L8). Not sure if this is why the install from repo fails, or if the lack of version attribute is related.

If I don't hear back I'll write a patch but it might be easier for someone who's already set up to have a squiz and test it. Thanks again.

Regards,
Tom

**Comment by andybotting** [comment]
Cheers for your great detective work. Looks like we have a mismatch between the add-on metadata and the metadata set in the repo.

I'll sort this out shortly and push an update.

Cheers,
Andy

**Comment by andybotting** [comment]
Should be fixed with 523edebba474bb9dfe19af2aee0270440e16cc1f

Thanks

**Comment by tkf144** [comment]
Can confirm the issue is resolved and the addon is installing from the repo once again. Thanks Andy.
